torch/_inductor/select_algorithm.py

```diff
@@ -380,7 +380,10 @@ def generate(
             )
             if self.debug:
                 print("Generated Code:\n", code)
-            mod = PyCodeCache.load(code)
+            mod = PyCodeCache.load(
+                code,
+                extra=f"{kwargs['BLOCK_M'], kwargs['BLOCK_N'], kwargs['BLOCK_K']}-{num_stages}-{num_warps}-",
```

Comment:
What if the config doesn't contain BLOCK_K -- or contains something else like BLOCK_G?

Reply:
I've never seen BLOCK_G, but I'll just make it dump all kwargs to be safe (although this will be much more noisy).

Reply:
Currently, it doesn't happen, but we want to future proof this code.